internal/store/memdb/memdb.go

```diff
@@ -300,6 +343,30 @@ func (memDb *MemDb) LoadFromGcs(ctx context.Context, bucket, prefix string) erro
 		}
 	}
 	log.Printf("Number of csv rows added: %d", count)
+	// Populate placeSvExistence field
```

Comment:
nit: This function is large enough... maybe this can be a helper with a useful name (like buildMemSVExistenceCache()?)